pynetdicom/service_class.py

```diff
@@ -1448,6 +1449,20 @@ def SCP(self, req, context):
                 evt.EVT_C_STORE,
                 {'request' : req, 'context' : context.as_tuple,}
             )
+            if req._dataset_file:
+                try:
+                    req._dataset_file.close()
+                except ValueError:
```

Comment:
The test doesn't seem to be raising an exception here

Reply:
My mistake, I misread this doc https://docs.python.org/3/library/io.html#io.IOBase.close - will remove this logic + its test